openfl-tutorials/experimental/Phi3/Workflow_Interface_Phi3.ipynb

```diff
@@ -0,0 +1,640 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "14821d97",
+   "metadata": {},
+   "source": [
+    "# Workflow Interface\n",
+    "## Fine-tuning Phi3 using OpenFL\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/intel/openfl/blob/develop/openfl-tutorials/experimental/Phi3/Workflow_Interface_Phi3.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bd059520",
+   "metadata": {},
+   "source": [
+    "In this tutorial, we build on the ideas from the [first](https://github.com/intel/openfl/blob/develop/openfl-tutorials/experimental/Workflow_Interface_101_MNIST.ipynb) quick start notebook, and demonstrate how to fine-tune a Large Language Model (LLM) in a federated learning workflow. \n",
+    "\n",
+    "We will fine-tune **Microsoft's [Phi3](https://huggingface.co/docs/transformers/main/en/model_doc/phi3)** model on the [Math_10k](https://github.com/AGI-Edgerunners/LLM-Adapters/tree/main) dataset, an open-source mathematical question-answer pair dataset collected from multiple smaller math datasets."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "39c3d86a",
+   "metadata": {},
+   "source": [
+    "# What is it?"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a7989e72",
+   "metadata": {},
+   "source": [
+    "The workflow interface is a new way of composing federated learning expermients with OpenFL. It was borne through conversations with researchers and existing users who had novel use cases that didn't quite fit the standard horizontal federated learning paradigm. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "124ae236-2e33-4349-9979-f506d796276d",
+   "metadata": {},
+   "source": [
+    "### Installing OpenFL\n",
+    "- Lets now install OpenFL and the necessary dependencies for the workflow interface by running the cell below:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c808dd12-6795-4203-9221-0f6b43fc785f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install git+https://github.com/intel/openfl.git\n",
+    "!cd /home/oamontoy/workspace/openfl/openfl-tutorials/experimental/Phi3 && pip install -r ../requirements_workflow_interface.txt\n",
```

Comment:
can you remove the absolute path and. keep general like 
!pip install git+https://github.com/securefederatedai/openfl.git
!pip install -r workflow_interface_requirements.txt